android-studio missing lsusb

### Describe the bug
android studio does not connect to usb devices because it does not `lsusb` as a runtime dependency.

### Steps To Reproduce
Steps to reproduce the behavior:
1. install android studio from `pkgs.android-studio`
1. open android studio in terminal
2. go to troubleshoot device connections (in dropdown to the left of the run button)
3. press rescan devices
4. check terminal output
```
2021-12-16 23:38:56,451 [  81447]   WARN - stUsbDevicesActionStateManager - java.io.IOException: Cannot run program "lsusb": error=2, No such file or directory
java.util.concurrent.CompletionException: java.io.IOException: Cannot run program "lsusb": error=2, No such file or directory
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1702)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1692)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.io.IOException: Cannot run program "lsusb": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at com.android.tools.usb.UsbDeviceCollectorImpl$execute$1.get(UsbDeviceCollector.kt:63)
	at com.android.tools.usb.UsbDeviceCollectorImpl$execute$1.get(UsbDeviceCollector.kt:62)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	... 6 more
Caused by: java.io.IOException: error=2, No such file or directory
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:340)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:271)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
	... 10 more
``` 

### Expected behavior
the derivation should include `lsusb`

### Screenshots
![image](https://user-images.githubusercontent.com/31556469/146465874-a6cc770c-971f-4edc-92cd-6bf2e2a79030.png)

### Additional context
Installing `usbutils` and repeating the steps fixes the issue

### Notify maintainers
@meutraa @fabianhjr 
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```
nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.7, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.5.0pre20211206_d1aaa7e`
 - channels(root): `"nixos-22.05pre338578.a2e281f5770"`
 - channels(prem): `"nixos-22.05pre335501.c71f061c68b"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`


```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/applications/editors/android-studio/common.nix
+++ b/pkgs/applications/editors/android-studio/common.nix
@@ -46,6 +46,7 @@
 , stdenv
 , systemd
 , unzip
+, usbutils
 , which
 , runCommand
 , xkeyboard_config
@@ -102,6 +103,7 @@ let
           # Runtime stuff
           git
           ps
+          usbutils
         ]}" \
         --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [
 

